Front-End/src/Components/Modals/Flagged.jsx

```diff
@@ -16,16 +16,16 @@ function Flagged(props) {
         results.data.forEach((video) => {
           video.comments.forEach((comment) => {
             array.push(comment);
-          })
-        })
+          });
+        });
         setFlagged(array);
       })
       .catch((err) => {
         console.log(err);
       });
   }, [modalShow]);
 
-  function handleDelete() {
+  function handleDelete(e) {
```

Comment:
yes! I have yet to write the query since it seems a bit complex because the comments are quite nested but that is the idea. Hopefully I can have time to do this after we get Kevin's video player working